ab2d-filters/src/main/java/gov/cms/ab2d/filter/EOBLoadUtilities.java

```diff
@@ -25,12 +25,15 @@ public class EOBLoadUtilities {
      *             Explanation of Benefit data retrieved from Blue
      * @return the ExplanationOfBenefit object
      */
-    public static org.hl7.fhir.dstu3.model.ExplanationOfBenefit getSTU3EOBFromFileInClassPath(String fileInClassPath) {
+    public static org.hl7.fhir.dstu3.model.ExplanationOfBenefit getSTU3EOBFromFileInClassPath(
+            String fileInClassPath) {
         if (StringUtils.isBlank(fileInClassPath)) {
             return null;
         }
-        try (InputStream inputStream = Thread.currentThread().getContextClassLoader().getResourceAsStream(fileInClassPath)) {
-            return FhirContext.forDstu3().newJsonParser().parseResource(org.hl7.fhir.dstu3.model.ExplanationOfBenefit.class, inputStream);
+        try (InputStream inputStream = Thread.currentThread().getContextClassLoader()
+                .getResourceAsStream(fileInClassPath)) {
+            return FhirContext.forDstu3().newJsonParser().parseResource(
+                    org.hl7.fhir.dstu3.model.ExplanationOfBenefit.class, inputStream);
```

Comment:
I personally have always been a fan of putting chained method calls on new lines. I've always found it easier to read and follow. Though, I wish the linter in this case did that for all of them rather than just based on a length requirement